articles/resources/relnotes.md

```diff
@@ -23,8 +23,9 @@ For update instructions, please refer to the [update guide](xref:microsoft.quant
 
 This release contains the following:
 
-- Improved IQ# debug user experience, to help developers better understand the inner workings of their quantum applications, and more easily identify logic errors. Created visualizations of a particular execution path of a quantum application, as well as the state at a point during the program execution.
-- Implemented a way for resource estimation to produce simultaneously achievable depth and width estimates.
+- Resource estimation now emits simultaneously achievable depth and width estimates in addition to the qubit count. See [here](xref:microsoft.quantum.machines.qc-trace-simulator.intro) for details.
```

Comment:
@bettinaheim I applied your suggestion, but from what I can see, that link doesn't seem to actually discuss this feature directly? Did you mean perhaps this link instead?

```suggestion
- Resource estimation now emits simultaneously achievable depth and width estimates in addition to the qubit count. See [here](xref:microsoft.quantum.machines.resources-estimator) for details.
```

Reply:
The best link would be to here: https://github.com/MicrosoftDocs/quantum-docs-pr/blob/main/articles/user-guide/machines/resources-estimator.md#metrics-reported

Reply:
No idea if `#` fragments work in these `xref` links, but let's give it a shot...

Reply:
Yes, it worked!